examples/headless_server/main.rs

```diff
@@ -0,0 +1,73 @@
+use amethyst::{
+    core::frame_limiter::FrameRateLimitStrategy,
+    ecs::{Join, System, WriteStorage},
+    network::*,
```

Comment:
nit: I'd rather not use a glob import like this

Reply:
That's how it was in the net_server example. =) I am going to leave it like this for consistency's sake. If we want to go back and remove all glob imports from the examples, we can do it as a different PR.